pkg/migration-tool/writer/writer.go

```diff
@@ -74,12 +88,24 @@ func (rw *RemoteWrite) Run(errChan chan<- error) {
 				if !ok {
 					return
 				}
-				blockRef.SetDescription("preparing to push", 1)
-				ts = timeseriesRefToTimeseries(blockRef.MergeProgressSeries(rw.progressTimeSeries))
-				buf = []byte{}
-				if err = rw.sendSamplesWithBackoff(context.Background(), ts, &buf); err != nil {
-					errChan <- fmt.Errorf("remote-write run: %w", err)
-					return
+				blockRef.UpdatePBarMax(blockRef.PBarMax() + rw.shardsSet.num)
```

Comment:
nit: shouldn't this be `blockRef.UpdatePBarMax(blockRef.PBarMax() + rw.shardsSet.num+1)` isn't there one extra step needed for the progressTimeSeries?

Reply:
Oh yes. Sorry, I missed that.